great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -159,14 +150,6 @@ def BaseDataContext(  # noqa: PLR0913
 
     --ge-feature-maturity-info--
     """
-    # Chetan - 20221208 - not formally deprecating these values until a future date
-    cloud_mode, cloud_config = _resolve_cloud_args(
-        cloud_mode=cloud_mode,
-        cloud_config=cloud_config,
-        ge_cloud_mode=ge_cloud_mode,
-        ge_cloud_config=ge_cloud_config,
-    )
-
```

Comment:
As part of this effort, we're removing any `ge_` prefixed args and related resolver methods - no need for formal deprecation since Cloud follows a different set of rules